### ISSUE TITLE
does not work on Gmail

### ISSUE THREAD

**Issue by dhruvkp** [akhodakivskiy/VimFx#118]
Hi, this is a great extension, but it is somehow not working for gmail website. I suppose there might be other websites too for which it doesn't work, as I think that it is the problem about keyboard focus. I wrote this issue on mozilla addon page too as a review, and you said that I should check the blacklist in preferences. I have checked the blacklist in extension's preferences. The blacklist is blank in my case, still it doesn't work. I am waiting for this issue to be resolved soon, as I use gmail web interface all the time for my emails. Thanks again for this great extension. :)

**Comment by prat0318** [comment]
Sorry for opening an old thread. But i am still facing this issue in the current version of gmail. None of the vimfx commands work in gmail (no matter if gmail's keyboard shortcuts are on or off). gmail is not in my vimfx's blacklist either.

This mostly causes an obstruction while i am switching tabs using Vimfx and suddenly when tab switch stops on reaching gmail tab :(
